base/event.jl

```diff
@@ -2,6 +2,39 @@
 
 ## condition variables
 
+
+if JULIA_PARTR
+
+import Core.Condition
+
+Condition() = ccall(:jl_condition_new, Ref{Condition}, ())
+wait(c::Condition) = ccall(:jl_task_wait, Cvoid, (Ref{Condition},), Ref{c})
+notify(c::Condition, arg, all, error) = ccall(:jl_task_notify, Cvoid, (Ref{Condition},), Ref{c})
```

Comment:
These should pass `c`, not `Ref{c}`. Same with `t` below. `Ref{c}` is a type, not a condition object.